packages/flutter/test/material/input_date_picker_form_field_test.dart

```diff
@@ -295,6 +308,17 @@ void main() {
       expect(containerColor, equals(Colors.transparent));
     });
 
+    testWidgets('DateFormat is followed when provided', (WidgetTester tester) async {
+      final GlobalKey<FormState> formKey = GlobalKey<FormState>();
```

Comment:
Looks like this property is not being used, could you remove it?

Reply:
Done :)